ci/scripts/run-test-network-basic.sh

```diff
@@ -0,0 +1,56 @@
+set -euo pipefail
+
+FABRIC_VERSION=${FABRIC_VERSION:-2.2}
+CHAINCODE_LANGUAGE=${CHAINCODE_LANGUAGE:-javascript}
+CHAINCODE_NAME=${CHAINCODE_NAME:-basic}
+
+function print() {
+	GREEN='\033[0;32m'
+  NC='\033[0m'
+  echo
+	echo -e "${GREEN}${1}${NC}"
+}
+
+print "Creating network"
+  ./network.sh up createChannel -ca -s couchdb -i "${FABRIC_VERSION}"
+print "Deploying ${CHAINCODE_NAME} chaincode"
+./network.sh deployCC -ccn "${CHAINCODE_NAME}" -ccv 1 -ccs 1 -ccl "${CHAINCODE_LANGUAGE}" -cci initLedger
```

Comment:
to Davids point in line 26: the app.js calls initLedger. If we keep that, no need for `-cci initLedger`